tests/test_rocksdb_options.rs

```diff
@@ -352,3 +353,14 @@ fn test_allow_concurrent_memtable_write() {
         db.put(format!("k_{}", i).as_bytes(), b"v").unwrap();
     }
 }
+
+#[test]
+fn test_get_compression() {
+    let mut opts = Options::new();
+    opts.create_if_missing(true);
+    opts.compression(DBCompressionType::DBSnappy);
+    match opts.get_compression() {
```

Comment:
`DBCompressionType` does't implement `std::cmp::PartialEq` and `std::fmt::Debug`, so cannot use `assert_eq!`.